### DISCUSSION TITLE
Show and Tell

### DISCUSSION THREAD

**Discussion by jamie-lynch** [guardian/cdk#173] [Show and tell]
Using `@guardian/cdk`? 🥇 

Tell us all about your experience 📘

**Comment by mbarton** [comment]
In Investigations & Reporting we're building a new tool powered by a lightweight API over a Postgres database. We decided to build it as a lambda backed by API gateway. We used the helpers from https://github.com/guardian/cdk/pull/28 in https://github.com/guardian/lurch/pull/15 and it worked first time 👏 

Using the CDK helpers from the team we got our app up and running in AWS in an afternoon. This would have taken at least a couple of days before. All time saved is vital for our team as we only have two fullt-time devs

**Comment by NovemberTang** [comment]
In TX we needed to create a very repetitive alarm stack. (nearly all identical except for one dimension) GuCDK meant we were able to write a couple of functions to autogenerate these alarms, plus set up a dashboard and keep all that infrastructure in the same place, down to the individual positioning of the dashboard widgets (which we can also dynamically calculate - this wouldnt have been possible if we were writing cloudformation by hand). Setting up new alarms is now as simple as a redeploy.
